server/src/send-update/send-update.controller.ts

```diff
@@ -24,15 +27,9 @@ export class SendUpdateController {
 
   @Post("/projects/:id/send-update")
   async sendUpdate(@Param() params, @Req() request: Request, @Res() response) {
-    const validatedUser = await this.listservAuthService.validateUser(request.headers.authorization)
-    if (!validatedUser) {
-      response.status(401).send();
-      return;
-    }
-
     const project = await this.projectService.findOneByName(params.id);
     // If no project is found, projectService returns HTTP error automatically, and this function does not continue
-    
+
     // Production names use underscores, staging use dashes
     var segments = project["data"]["attributes"]["dcp-borough"] === "Citywide" ? [{ name: "CW", envSegment: `zap${this.sendgridEnvironment === "production" ? "_production_" : "-staging-"}CW`, segmentId: "" }] : [];
```

Comment:
I just asked Pratishta to update the staging segments names to use underscores like production so go ahead and update this accordingly.